docs/SOPs/dataset_wrangling_SOP.md

```diff
@@ -124,30 +124,34 @@ Once you have a customised and potentially pre-filled spreadsheet it can be sent
 
 ### AWS User for Contributors
 
-In order for a contributor to upload their data, they would need their own AWS user that is assigned to the hca-contirbutor group. 
-A request for an account should be files as [a new ticket](https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/new?assignees=&labels=operations&template=new-contributor-account.md&title=contributor+account+for%3A+%3Ccontributor-name%3E) for the ingest team. The board is monitored regularly so the new ticket would be picked up within the day.
+In order for a contributor to upload their data, they would need their own AWS user that is assigned to the hca-contirbutor group.
```

Comment:
Please keep the part about opening a ticket using the "new contributor account" template.

Reply:
Ah, sorry, that was not intentional. I am going to put it back.

Reply:
Done